server

```diff
@@ -127,6 +127,17 @@ query_default_version() {
   DEFAULT_SCYLLA_VERSION=$(echo $DEFAULT_SCYLLA_VERSION_RAW | sed -e "s/.*version\":\"\(.*\)\".*/\1/g")
 }
 
+packages_update () {
+  OS_TYPE=$(cat /etc/os-release | awk -F '=' '/^NAME/{print $2}' | awk '{print $1}' | tr -d '"')
```

Comment:
we are running  ` . /etc/os-release` in line 277

Reply:
Right! Fixed. This line was removed and the lines of ". /etc/os-release" moved before calling the function